src/layouts/Layout.vue

```diff
@@ -50,17 +50,24 @@ export default {
     isCoach () {
       return COACH_ROLES.includes(this.user.role.name);
     },
+    isHelper () {
+      return this.user.role.name === HELPER;
+    },
     menuIcon () {
       return this.isMini ? 'view_headline' : 'chevron_left';
     },
     toggleDrawer () {
       return this.$store.state.main.toggleDrawer;
     },
     sidemenusRefs () {
-      if (this.user && !this.isAuxiliary) {
-        return 'defaultMenu';
+      if (this.user && this.isAuxiliary) {
+        return 'auxiliaryMenu';
+      } else if (this.user && this.isCoach) {
+        return 'coachMenu'
+      } else if (this.user && this.isHelper) {
+        return 'helperMenu';
```

Comment:
tu peux ajoutes au debut un `if (!this.user) return 'defaultMenu` comme ca tu n'as plus a verifier le cas
et tu n'as pas besoin des else-if puisque tu retournes direct